sdk/cargo-test-bpf/src/main.rs

```diff
@@ -65,6 +69,22 @@ where
 
     let exit_status = child.wait().expect("failed to wait on child");
     if !exit_status.success() {
+        eprintln!("CARGO-TEST-BPF exited on command execution failure.");
+        if !generate_rerun {
+            exit(1);
+        }
+        let rerun = File::create("rerun-script.sh").unwrap();
```

Comment:
"rerun" doesn't feel like the right nomenclature here. "spawn" perhaps, or "child"? 
Also we should namespace the script since it gets dumped into the cwd, so perhaps call it `cargo-test-bpf-child-script.sh`.  Even better maybe `format!("cargo-test-bpf-child-script-{}.sh", program.display())` (but strip the path) since `spawn()` invokes multiple programs.

There's also a duplicate `spawn()` in `cargo-build-bpf` that could benefit from this same feature in the future perhaps